lib/iris/cube.py

```diff
@@ -41,7 +41,7 @@
 import iris._concatenate
 import iris._constraints
 from iris._data_manager import DataManager
-from iris._lazy_data import lazy_elementwise
+from iris._lazy_data import lazy_elementwise, co_realise_cubes
```

Comment:
Exposing this as ```iris.cube.co_realise_cubes``` isn't ideal.

Reply:
It isn't exposed, because it's not in the `__all__`.
no?

Reply:
Importing into the module namespace make it available there from outside, the `__all__` list only affects `from iris.cube import *`.

Instead of importing and calling `co_realise_cubes` you could take the few lines of code that define `co_realise_cubes` and put them in the `realise_cubes` method, and have that as the  only interface to the functionality (deleting the `co_realise_cubes` altogether instead of calling it from `realise_cubes`).

Reply:
> @ajdawson can we import just "iris._lazy_data" here, instead of specific named components ?

New commit attempts to do this...